lang/java/reef-io/src/main/java/org/apache/reef/io/network/naming/NameServerImpl.java

```diff
@@ -96,44 +93,6 @@ public NameServerImpl(
     LOG.log(Level.FINE, "NameServer starting, listening at port {0}", this.port);
   }
 
-  /**
-   * @deprecated have an instance injected instead
-   */
-  @Deprecated
-  public NameServerImpl(final int port, final IdentifierFactory factory) {
```

Comment:
We need to keep all the old constructors in 0.12. Please update the JavaDoc as per the above, though.